examples/angular/src/app/pages/wallet-selector-export/wallet-selector-export.component.ts

```diff
@@ -81,6 +82,12 @@ export class WalletSelectorExportComponent implements OnInit {
         }),
         setupRamperWallet(),
         setupNearMobileWallet(),
+        setupMintbaseWallet({
+          networkId: "testnet",
+          walletUrl: "https://testnet.wallet.mintbase.xyz",
+          callbackUrl: "http://localhost:4200",
+          deprecated: false,
```

Comment:
@kujtimprenkuSQA  our team will fix the issues ASAP

Reply:
@kujtimprenkuSQA  made all params optional now. We want the user to be able to pass the params because there are some scenarios like: 

1. getting the network from the website url
2. global callbackUrls
3. wallet url can be tested on localhost by Mintbase team or in one of our dev/staging servers, so sometimes we add dynamic wallet urls.